llm/auto_parallel/llama/run_pretrain_auto.sh

```diff
@@ -74,6 +74,7 @@ python -u  -m paddle.distributed.launch \
     --do_eval \
     --device "gpu" \
     --data_impl "mmap" \
+    --replace_with_parallel_cross_entropy 0 \
```

Comment:
Would it be better to place `replace_with_parallel_cross_entropy`  in `tensor_parallel_config` configuration?

Reply:
Done
Move `replacew_ith_marallel_cross_detropy ` to `tensor_marallel_config ` for configuration.